src/status_im/core.cljs

```diff
@@ -62,7 +62,7 @@
       (.addListener ^js react/keyboard
                     (if platform/ios?
                       "keyboardWillHide"
-                      "keyboardWDidHide")
+                      "keyboardDidHide")
```

Comment:
so it didn't work before ?

Reply:
No, on android after first open it was using always height as "open"